docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -16,16 +13,133 @@ This document covers a range of common scenarios using the PXIe-148X Generation
+    - Set the **timestamp filter** array to contain only the **User24** timestamp ID. This will let you see the I2C traffic on SI0. If you run the VI with a configuration script selected, you will see configuration traffic in the I2C Data Output tab after the VI has stopped.
+
+    ![Configuration Settings I2C Tab](../../images/PXIe-148X-Acq-I2CTab-FiniteAcq.png)
+
+5.  Run the Acquisition VI. 
+
+6.  Run the Generation VI.
+
+7.  Select the **I2C Timestamps** tab to view I2C timestamp data.
+    - View the displayed I2C timestamp data in the **I2C Timestamps** table. The **I2C Timestamps** table displays I2C timestamp information for all I2C traffic. I2C timestamps begin logging immediately after the FPGA bitfile is downloaded and include timestamp data prior to the start of the LLP packet data acquisition (i.e. I2C traffic from the configuration script).
 
-## Generating and Filtering LLP Packets
+        > Note: To display I2C timestamp data, **Log I2C to Disk** must be enabled and desired timestamp IDs must be added to the **timestamp filter** array. The I2C timestamp data displayed is read from the User_Timestamps.tdms file and filtered to display only timestamp IDs included in the timestamp filer array. The **I2C Timestamps** display is updated after the acquisitions completes.
+
+    ![I2C Timestamps Data](../../images/PXIe-148X-Acq-I2CTimestamps-FiniteAcq.png)
 
 ## Playing Back Previously Acquired Data
+1. Refer to the Commmon Acquisition Tutorial section Acquiring and Filtering LLP Packets (TODO LINK) to acquire LLP packets from a camera.
+ 
+2. Double click the Generation Example VI in the generation LabVIEW project.
+
+3. Set the following controls on the Generation Example GSE VI and leave all other values at their defaults.
+    > Note: VI controls and indicators can be reset to default values by clicking on the **Edit** menu and selecting the **Reinitialize Values to Default** option.
+
+    | Tab | Control | Value |
+    |---|---|---|
+    | Resource | RIO Device | [System Specific] |
+    | Resource | Bitfile Path | [Refer to Bitfile Path in the PXIe-148X Generation GSE Help](../../reference/gettingstartedexample/gse-gen-help.md#table-of-pxie-148x-generation-bitfiles) |
+    | Resource | TDMS File Directory| A directory path to TDMS files containing LLP Packet data for serial output channels (e.g. <font face = "courier new">SI0_LLP_Packets.tdms</font>) |
+    
+    > Note: The **TDMS File Directory** is a folder selection and the browse dialog shows folders only, not file names. 
+    >
+
+4. Run the Generation Example VI.
+
+5. Select the **First Display Channel** tab on the Generation Example and verify that images from the previously acquired data are displayed on this tab. 
```

Comment:
I think we should maybe take a step back and think about what the intent of this section is - what do we want the reader to learn? I'm not sure that this is teaching them anything that they don't already know?

Reply:
Yeah I had that thought too, when I reduced it down to just linking to the acq tutorial. It still might just serve as a way for people to realize that all the TDMS files are compatible between acq and gen.